docs/src/example_networks/hodgkin_huxley_equation.md

```diff
@@ -0,0 +1,182 @@
+\frac{0.6000000000000001 + 0.010000000000000002 V}{1 - e^{-6 + \frac{-1}{10} V}} & \text{otherwise}
+\end{cases}$][$0.125 e^{\frac{-7}{8} + \frac{-1}{80} V\left( t \right)}$] n}\\
+\ce{ m^\prime &<=>[$\begin{cases}
+1.0 & \text{if } \left( \frac{9}{2} + \frac{1}{10} V = 0.0 \right)\\\\\\
+\frac{\frac{9}{2} + \frac{1}{10} V}{1 - e^{\frac{-9}{2} + \frac{-1}{10} V}} & \text{otherwise}
+\end{cases}$][$4 e^{\frac{-35}{9} + \frac{-1}{18} V\left( t \right)}$] m}\\\\\\
+\ce{ h^\prime &<=>[$0.07 e^{\frac{-7}{2} + \frac{-1}{20} V}$][$\frac{1}{1 + e^{-4 + \frac{-1}{10} V\left( t \right)}}$] h}
+\end{align*}
+$$
+
+</br>
+
+Next we create a `ModelingToolkit.ODESystem` to store the equation for `dV/dt`
+
+```julia
+voltageode = let
+	@parameters C=1.0 ḡNa=120.0 ḡK=36.0 ḡL=.3 ENa=45.0 EK=-82.0 EL=-59.0 I₀=0.0
+	@variables m(t) n(t) h(t)
+	I = I₀ * sin(2*pi*t/30)^2 
+
+	Dₜ = Differential(t)
+	eqs = [Dₜ(V) ~ -1/C * (ḡK*n^4*(V-EK) + ḡNa*m^3*h*(V-ENa) + ḡL*(V-EL)) + I/C]
+	@named voltageode = ODESystem(eqs, t)
+end
+```
+
+Output:
+
+$voltageode =$
+
+$$
+\begin{align}
+\frac{dV(t)}{dt} =& \frac{ - {\textrm{\={g}}}L \left(  - EL + V\left( t \right) \right) - \left( n\left( t \right) \right)^{4} {\textrm{\={g}}}K \left(  - EK + V\left( t \right) \right) - \left( m\left( t \right) \right)^{3} {\textrm{\={g}}}Na \left(  - ENa + V\left( t \right) \right) h\left( t \right)}{C} + \frac{\sin^{2}\left( 0.20943951023931953 t \right) I_0}{C}
+\end{align}
+$$
+
+</br>
+
+Notice, we included an applied current, `I`, that we will use to perturb the system and create action potentials. For now we turn this off by setting its amplitude, `I₀`, to zero.
+
+Finally, we couple this ODE into the reaction model as a constraint system:
+
+```julia
+@named hhmodel = extend(voltageode, hhrn);
+```
+
+* `hhmodel` is now a `ReactionSystem` that is coupled to an internal constraint `ODESystem` that stores `dV/dt`.
+
+* Let's now solve to steady-state, as we can then use these resting values as an initial condition before applying a current to create an action potential.
+</br>
+
+```julia
+hhsssol = let
+	tspan = (0.0, 50.0)
+	u₀ = [:V => -70, :m => 0.0, :h => 0.0, :n => 0.0, 
+		  :m′ => 1.0, :n′ => 1.0, :h′ => 1.0]
+	oprob = ODEProblem(hhmodel, u₀, tspan)
+	sol = solve(oprob, Rosenbrock23())	
+end;
+```
+
+```julia
+plot(hhsssol, vars=V)
+```
+Output:<br/>
+![Plot1](../assets/hogkin_huxley_plot1.svg)
+
+```julia
+u_ss = hhsssol.u[end]
+```
+Output:</br>
+`u_ss = [0.680779, 0.319221, 0.946472, 0.0535277, 0.408161, 0.591839, -69.9048]`
+</br>
+
+Finally, starting from this resting state let's solve the system when the amplitude of the stimulus is non-zero and see if we get action potentials
+
+```julia
+let
+	tspan = (0.0, 50.0)
+	@unpack I₀ = hhmodel
+	oprob = ODEProblem(hhmodel, u_ss, tspan, [I₀ => 10.0])
+	sol = solve(oprob)
+	plot(sol, vars=V, legend=:outerright)
+end
+```
+</br>
+Output:</br>
+
+![Plot2](../assets/hogkin_huxley_plot2.svg)
```

Comment:
Let the example blocks generate the figures instead of including pre-generated ones.